### ISSUE TITLE
processor info displays blank for system z

### ISSUE THREAD

**Issue by sureshab** [kimchi-project/kimchi#728]
Architecture - s390
Current kimchi/kimchi-wok doesn't provide any information specific to z architecture. 
![screenshot from 2015-09-07 16 48 38](https://cloud.githubusercontent.com/assets/12499230/9715772/5a92b798-5581-11e5-98f7-9db10fe4e777.png)

**Comment by sureshab** [comment]
I am working on adding some of z architecture specific information in basic information section(/plugins/kimchi/host)

**Comment by ramonmedeiros** [comment]
I can take a look. Please, send me the machine access info by mail:

ramonn@linux.vnet.ibm.com

**Comment by sureshab** [comment]
Hi Ramon,
As explained in the email, i would be working on this issue, also adding additional s390 architecture specific information. Please let me know if you have any concerns.
Patch will be submitted once gingerbase is merged into wok, just to avoid duplication since the host basic information is moving to gingerbase plugin.

**Comment by alinefm** [comment]
Hi @sureshab!
I tagged this issue as a feature as Kimchi does not has properly support for Z systems.
Please, let me know if you are going to do that for Kimchi 2.0, so I can properly set the milestone.

**Comment by sureshab** [comment]
Hi @alinefm!
sure, please add this feature as part kimchi 2.0 milestones.

**Comment by alinefm** [comment]
Fixed by d8445ce1
